Feat/new transactions page

adrianmcli opened the pull request:
**Note:** Please merge #904 first.

![screen shot 2018-09-16 at 6 37 52 pm](https://user-images.githubusercontent.com/943555/45603294-c3e1c980-b9df-11e8-8494-1b123bde02df.png)

^ this is what it looks like now

### What has been done:

- Added new section for Contract Info
- Added new section for Events
- Added hardcoded data for now, should be simple to wire up afterwards

### Of note:

I was not sure about the structure of the Contract Info and Events, but I made simple placeholder objects so that we can easily wire it up in the future.

seesemichaelj commented:
I receive this error in the console when I opened up the transaction detail page:
```
/home/mike/work/consensys/ganache/node_modules/react/cjs/react.development.js:217 Warning: Each child in an array or iterator should have a unique "key" prop.

Check the render method of `TxCard`. See https://fb.me/react-warning-keys for more information.
    in div (created by TxCard)
    in TxCard (created by Connect(TxCard))
    in Connect(TxCard) (created by TransactionsScreen)
    in div (created by TransactionsScreen)
    in TransactionsScreen (created by Connect(TransactionsScreen))
    in Connect(TransactionsScreen) (created by RouterContext)
    in div (created by Scrollbars)
    in div (created by Scrollbars)
    in Scrollbars (created by AppShell)
    in div (created by AppShell)
    in div (created by AppShell)
    in AppShell (created by Connect(AppShell))
    in Connect(AppShell) (created by RouterContext)
    in RouterContext (created by Router)
    in Router
    in Provider
    in AppContainer
```

adrianmcli commented:
![screen shot 2018-09-17 at 10 27 08 am](https://user-images.githubusercontent.com/943555/45639240-52982a00-ba64-11e8-8923-d755856c7c89.png)

@seesemichaelj I've fixed the errors and updated the Contract Info data. Is this PR good to merge now?
